apps/dapp/src/services/AnalyticsService.ts

```diff
@@ -0,0 +1,35 @@
+import env from 'constants/env';
+import posthog from 'posthog-js';
+import { isDevelopmentEnv } from 'utils/helpers';
+
+export class AnalyticsService {
+  private static instance: AnalyticsService;
+  private static shouldSendAnalytics: boolean;
+
+  private constructor() {
+    AnalyticsService.shouldSendAnalytics = !isDevelopmentEnv();
+  }
+
+  public static getInstance(): AnalyticsService {
+    if (!AnalyticsService.instance) {
+        AnalyticsService.instance = new AnalyticsService();
```

Comment:
Is there a need to create an instance vs having everything be static?

Reply:
I'll ditch the singleton.

Reply:
https://github.com/TempleDAO/temple/pull/486/commits/59ec0703e6bc95876708be100bdcdef4e74ceb5d